_gsocproposals/2021/proposal-ATLASCompressionAE.md

```diff
@@ -51,7 +51,7 @@ An improved autoencoder network with documentation and figures of merit that may
   * [PyTorch](http://pytorch.org)
 
  ## Mentors
-  * [Caterina Doglioni](mailto:caterina.doglioni@cern.ch)
+  * [Caterina Doglioni](mailto:caterina.doglioni@cern.ch) [main]
```

Comment:
Instructions are to put the main mentor in bold letters ( * **[Caterina Doglioni](mailto:caterina.doglioni@cern.ch)**)